scripts/build/builders/k32w.py

```diff
@@ -18,6 +18,35 @@
 from .gn import GnBuilder
 
 
+class K32WBoard(Enum):
+    K32W0 = auto()
+    K32W1 = auto()
+
+    def AppName(self):
+        if self == K32WBoard.K32W0:
+            return 'k32w0x'
+        elif self == K32WBoard.K32W1:
+            return 'k32w1'
+        else:
+            raise Exception('Unknown board type: %r' % self)
+
+    def FolderName(self):
+        if self == K32WBoard.K32W0:
+            return 'k32w0'
+        elif self == K32WBoard.K32W1:
+            return 'k32w1'
+        else:
+            raise Exception('Unknown board type: %r' % self)
+
+    def Extensions(self):
```

Comment:
Could you add a comment on what this iss supposed to return? Name does not seem as intuitive - extensions of what and why is blank a valid extension?

Would it makes sense to return just the hex/srec as some `BinaryTextExtension`?

Reply:
I didn't add a comment, but I renamed the method and slightly updated the implementation. I hope it's not confusing anymore. The extension is different on the two platforms due to the different tools we are using: MCUXpresso with CMSIS-DAP/JLink.

Reply:
Never mind. I will remove the extension and keep only the `.elf` and `.map` extensions. The previous `blank` extension was the actual elf file.